### PR TITLE
Issue #164 : Added project dependency list

### PR DESCRIPTION
**Pull request by IamAdiSri**
#164

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -93,6 +93,20 @@ pip install -r requirements.txt
...
+StopStalk is built on the [Web2Py Framework](http://www.web2py.com), which is a Python based MVC framework.
+The project also depends on a number of other open source packages, some of which are 
+
+- [MySQL](http://www.mysql.com)
+- [Google Calender API](https://developers.google.com/google-apps/calendar/)
+- [Google Charts API](https://developers.google.com/chart/)
+- [Python Requests Library](http://docs.python-requests.org/en/master/)
+- [Beautiful Soup](https://www.crummy.com/software/BeautifulSoup/)
+
+If you are a new to open source contributing or want to learn more about this project, please feel free to go through the links above.
```
Comment (raj454raj) [inline README.md]: If you are new

File: README.md [hunk 2]
```diff
@@ -93,6 +93,20 @@ pip install -r requirements.txt
...
+Contribution guidelines are also given below.
```
Comment (IamAdiSri) [inline README.md]: @raj454raj  Should i remove line 107 and 108?

File: README.md [hunk 3]
```diff
@@ -104,7 +104,7 @@ The project also depends on a number of other open source packages, some of whic
 - [Python Requests Library](http://docs.python-requests.org/en/master/)
 - [Beautiful Soup](https://www.crummy.com/software/BeautifulSoup/)
 
-If you are a new to open source contributing or want to learn more about this project, please feel free to go through the links above.
+If you are inexperienced at contributing to open source or want to learn more about this project, please feel free to go through the links above.
```
Comment (raj454raj) [inline README.md]: Well actually you might remove this line altogether.

**Comment by raj454raj** [comment]
Thank you @IamAdiSri ! :+1:

**Pull request merged**